internal/pkg/workspace/workspace.go

```diff
@@ -177,8 +177,25 @@ func (ws *Workspace) workloadNames(match func(string) bool) ([]string, error) {
 	return names, nil
 }
 
-// ReadWorkloadManifest returns the contents of the workload's manifest under copilot/{name}/manifest.yml.
-func (ws *Workspace) ReadWorkloadManifest(name string) ([]byte, error) {
+// ReadServiceManifest returns the contents of the service's manifest under copilot/{name}/manifest.yml.
+func (ws *Workspace) ReadServiceManifest(name string) ([]byte, error) {
+	mf, err := ws.readWorkloadManifest(name)
+	if err != nil {
+		return nil, fmt.Errorf("read manifest file %s: %w", name, err)
+	}
+	return mf, nil
+}
+
+// ReadJobManifest returns the contents of the job's manifest under copilot/{name}/manifest.yml.
+func (ws *Workspace) ReadJobManifest(name string) ([]byte, error) {
+	mf, err := ws.readWorkloadManifest(name)
+	if err != nil {
+		return nil, fmt.Errorf("read manifest file %s: %w", name, err)
```

Comment:
Can we either reword these errors to reference services and jobs, or just use a single workload-agnostic function like we did with `relativeDockerfilePath`?

```suggestion
		return nil, fmt.Errorf("read manifest file for job %s: %w", name, err)
```

Reply:
I'll change them to more closely align with the writeWorkloadManifest error messages. Thanks!